samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages.properties

```diff
@@ -595,6 +595,7 @@ partial_credit_limit_detail=Value needs to be a whole number between 0 and 99.
 edit_published_assessment_warn_1=This assessment has been retracted from student view.  
 edit_published_assessment_warn_21=Click "Republish" or "Regrade and Republish" to make it available to students.
 edit_published_assessment_warn_22=Click "Republish" to make it available to students.
+edit_published_assessment_warn_3=<strong>If you do not want to remove the previous submissions DO NOT check the checkbox option on the next screen.</strong>
```

Comment:
too vague. what checkbox option?

Reply:
I think they are referring to the checkbox in the next screen, I also had doubts about this patch's utility.

Reply:
@ottenhoff I'm closing this, the current UI is more than enough to warn instructors. I believe this patch comes from many years ago where that information was not present and was no confirmation.

![image](https://github.com/sakaiproject/sakai/assets/8653754/e8ba2d1a-5e9d-4562-a4ff-5d0d1025145e)